samples/script/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM sk8s/shell-function-invoker:latest
+ADD *.sh /functions/script/
```

Comment:
this should also just be explicit about `wordcount.sh` for now. The goal as a follow-up will be to have a default file location/name so that the `FUNCTION_URI` env var is not needed for the pre-built immutable container cases.